apps/src/utils.js

```diff
@@ -950,3 +950,12 @@ export function isTestEnvironment() {
 export function isProductionEnvironment() {
   return getEnvironment() === Environments.production;
 }
+
+/**
+ * Fetch cookies signed by cloudfront which grant access to restricted content.
+ * @returns {Promise<Response>}
+ * TODO: Reuse this in Dance Party (Dance.js and songs.js)
```

Comment:
This function is copied directly from [dance party code](https://github.com/code-dot-org/code-dot-org/blob/d499421eeff5100d55c2543b7900d66be1e81bfd/apps/src/dance/songs.js#L47). Ideally, we should reuse this so it stays consistent across labs, but I opted to leave this as a TODO for now to minimize impact outside of music lab.